### PR TITLE
Issue #27 - Add basic CRUD operation step definitions

### LINKED ISSUE #27: Add step definitions for block_content module
Add CRUD events support for `block_content`.

### PR DESCRIPTION
**Pull request by balintpekker**
Add CRUD step definitions for block_content.
Todo: blockContentCreate() method.

Closes [#27](https://github.com/Cheppers/drupal-behat-contexts/issues/27)

### CODE REVIEW

File: src/Context/Drupal/BlockContentContext.php [hunk 1]
```diff
@@ -0,0 +1,467 @@
...
+use Webmozart\Assert\Assert;
+
+class BlockContentContext extends Base
+{
+    /**
+     * Keep track of block contents so they can be cleaned up.
+     *
+     * @var array
+     */
+    protected $blockContents = [];
```
Comment (Sweetchuck) [inline src/Context/Drupal/BlockContentContext.php]: \Cheppers\DrupalExtension\Context\Drupal\AppContentEntitySetupTearDown

File: src/Context/Drupal/BlockContentContext.php [hunk 2]
```diff
@@ -0,0 +1,467 @@
...
+     *
+     * @param string $type
+     *   Block content type machine name.
+     * @param string $info
+     *   Block content info.
+     *
+     * @Given I am visiting the :type block content :info
+     * @Given I visit the :type block content :info
+     */
+    public function iAmViewingTheBlockContent(string $type, string $info)
```
Comment (Sweetchuck) [inline src/Context/Drupal/BlockContentContext.php]: Naming convention of methods:
* Method which creates or do something (eg: click on a link) has to start with `do`.
* Method which checks something has to start with `assert`.

See examples in the existing Context classes (./src/Context/Drupal/*)
* \Cheppers\DrupalExtension\Context\Drupal\CoreTabs::assertPrimaryTabsTable()
* \Cheppers\DrupalExtension\Context\Drupal\CoreTabs::doClickPrimaryTab()

File: src/Context/Drupal/BlockContentContext.php (hunk 1 above)
Comment (balintpekker) [reply src/Context/Drupal/BlockContentContext.php]: Removed in commit 6434a92.

File: src/Context/Drupal/BlockContentContext.php (hunk 2 above)
Comment (balintpekker) [reply src/Context/Drupal/BlockContentContext.php]: Fixed in commit 6434a92.

**Pull request closed without merging**